docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
-* To create the new discount, click **Save**.
+The following table describes the attributes you enter and select on the **General information** tab:
+
+<div class="width-100">
+
+| ATTRIBUTE |DESCRIPTION  |
+| --- | --- |
+| STORE RELATION | Stores that your cart rule will be active in. You can select multiple stores.|
+| DISCOUNT TYPE | Drop-down list where you select either **Voucher codes** or **Cart rule** discount type. |
+| NAME | Unique name that will be displayed in the cart, in the **Available discounts** section on the Storefront, if the cart rule is valid. Should be short, but descriptive. |
+| DESCRIPTION | Explains the voucher and helps a customer understand why they are eligible for the discount and what they can receive. The description will be displayed in the **Cart calculation** section along with the applied discount amount if the discount is applicable to a product in the cart.|
+| PRIORITY | Defines [the discount priority](/docs/scos/user/features/{{page.version}}/promotions-discounts-feature-overview.html#discount-priority). Represented as an integer value from `1` to `9999`, `1` being the highest priority and `9999` the lowest. |
+| NON-EXCLUSIVE | Defines the discount exclusivity. Customers can redeem non-exclusive discounts in conjunction with other non-exclusive discounts.|
+| EXCLUSIVE | Defines the discount exclusivity. When a discount is exclusive, no other discounts may be applied in conjunction. When a cart is eligible for multiple exclusive discounts, the discount with the highest value to the customer is applied. The exception to this is promotional product discounts. Query string discounts and promotional product discounts exclude only among each other. Promotional product discounts are not affected by exclusive query string discounts and conversely.|
+| VALID FROM and VALID TO | Cart rule will be active between **VALID FROM** and **VALID TO** dates and times (in UTC), inclusive. For example, a discount applies to the cart starting from 01.12.2021 23:00 until 31.01.2022 22:59, UTC. |
+
+</div>
+
+{% info_block infoBox "Info" %}
+
+Name and describe discounts in a way that is meaningful for other Back Office users. Besides, the given name is displayed in the customer's cart. Therefore, it must be unique.
+
+{% endinfo_block %}
+
+#### Discount calculation tab
+
+This section contains information you need to know when working with discount calculations on the **Discount calculation** tab.
+
+
+**CALCULATOR TYPE**
+<br>The discount can be calculated in two ways:
+* **Percentage**: A discount is calculated as a percentage of the discounted items' prices. If selected, in the **VALUE** field, set the percentage value (for example, `25`).
+* **Fixed amount**: A fixed amount is discounted. If you select this type, specify the amount (**Gross price**, **Net price**, or both) for each currency used in your store.
```

Comment:
Same as the previous.

Reply:
Also, check the reset of the doc for parentheses.

Reply:
Done